examples/prism-rescript.html

```diff
@@ -0,0 +1,57 @@
+<h2>Comments</h2>
+<pre><code>/* This is a comment */
+// Another comment
+</code></pre>
+
+<h2>Let bindings</h2>
+<pre><code>let name = "Alonzo Church"
+let message = `Hello ${name}`
+let message2 = `Hello ${person.name}`
+let result = `Testing => ${Module.Another.value}`
+let value = 1
+let result = 1 + 1
+let langs = ["ReScript", "JavaScript"]
+let person = ("Alonzo", 32)
+</code></pre>
+
+<h2>Type declarations & Functions</h2>
+<pre><code>type role = | Admin | Editor | Viewer 
+type numeric = #1 | #2 | #3
+type normal = #Poly | #Variant
+type myPolyVar = [ #"poly-variant" | #"test-chars" ]
+
+type person = {
+ name: string,
+ age: int,
+ role: role
+}
+
+type record = {
+  "field": string
+}
+
+let sum = (a, b) => a + b
+let sum2 = (a:int, b: int) => a + b
+</code></pre>
+
+<h2>Modules, JSX & Components</h2>
+<pre>
```

Comment:
Test currently fail because of the new line here. There must not be any text between `pre` and `code`.

Make it `<pre><code>...</code></pre>` everywhere and the tests for example will pass.

Reply:
Oh, great!

Reply:
Done